stripe/src/main/java/com/stripe/android/view/CardFormView.kt

```diff
@@ -430,14 +430,14 @@ internal class CardFormView @JvmOverloads constructor(
             1
         )
 
-        // add horizontal divider above postalCodeContainer and hide countryPostalDivider
-        postalCodeContainer.addView(
+        // add horizontal divider below countryLayout and hide countryPostalDivider
+        countryLayout.addView(
             StripeHorizontalDividerBinding.inflate(
                 layoutInflater,
                 countryLayout,
                 false
             ).root,
-            0
+            1
```

Comment:
If it's the last child we don't need an index

Reply:
You can also use the ktx, 

```
countryLayout += StripeHorizontalDividerBinding.inflate(
                layoutInflater,
                countryLayout,
                false
            ).root
```

Reply:
oh sorry missed the 2nd comment, will keep calling same functions now as there're other calls where we don't necessary add to the last child